library/general/test/report_test.rb

```diff
@@ -121,4 +124,58 @@
       expect(subject.GetMessages(0, 1, 0, 0)).to match(/Message/)
     end
   end
+
+  describe ".Settings" do
```

Comment:
Hm, not only Export but also default settings while normal installation. True, I am using Export for checking the values :-)